dipy/reconst/peak_direction_getter.pyx

```diff
@@ -38,6 +38,10 @@ cdef class PeaksAndMetricsDirectionGetter(DirectionGetter):
         self.ang_thr = 60
         self.total_weight = .5
 
+    def __getstate__(self): return self.__dict__
+
+    def __setstate__(self, d): self.__dict__.update(d)
```

Comment:
The `update` will leave any previous members in place, that are not defined in `d`.  Is that the right thing to do?